### PR TITLE
package xtl for conan-io

### PR DESCRIPTION
**Pull request by yssource**
Specify library name and version:  **xtl/0.6.10**

This PR is required by #541

- [ x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/xtl/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from conans import ConanFile, tools
+
+
+class XtlConan(ConanFile):
+    name = "xtl"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtl"
+    description = "The x template library"
+    topics = ("conan", "templates", "containers", "algorithms")
```
Comment (danimtb) [inline recipes/xtl/all/conanfile.py]:
```suggestion
    topics = ("conan", "templates", "containers", "algorithms")
    settings = "os", "compiler", "arch", "build_type"
```

Comment (uilianries) [reply recipes/xtl/all/conanfile.py]: Why excluding package_id instead? We don't need setting in this recipe, it's only copying the files to the package folder, no cmake interaction.

Comment (danimtb) [reply recipes/xtl/all/conanfile.py]: either one approach or the one you say would be fine

Comment (yssource) [reply recipes/xtl/all/conanfile.py]: "excluding package_id" makes sense to me.

**Pull request merged**